_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,425 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.fieldset {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .group-legend {
+        margin-bottom: spacing-vertical(x-small);
+    }
+
+    .field-wrapper {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $helper-text;
+        }
+    }
+
+    &.form-group {
+
+        .field-wrapper { // Generally used for radios and checkboxes
+            margin-bottom: spacing-vertical(small);
+        }
+
+        .helper-text { // Generally used for radios and checkboxes
+            margin-top: spacing-vertical(xx-small);
+            margin-left: 24px;
+        }
+    }
+}
+
+.form-label {
+    display: block;
+    margin-bottom: spacing-vertical(x-small);
+    font-size: font-size(mid-large);
+    color: $label-color;
+
+    &.inline-label {
+        margin-bottom: 0;
+    }
+
+    &.is-active,
+    &.is-selected {
+        color: $label-color-active;
+    }
+}
+
+.helper-text {
+    margin-top: spacing-vertical(x-small);
+}
+
+.required-field:after {
+    display: inline-block;
+    margin-left: ($baseline/4) + px;
+    content: "*";
+    font-size: font-size(small);
+    color: $required-indicator-color;
+}
+
+.input,
+.select,
+.textarea {
+    @include transition(all timing(xx-fast) ease-in-out 0s);
+    display: inline-block;
+    min-width: ($baseline*20) + px; // for the purposes of the demo... can probably get moved elsewhere...
```

Comment:
Also, it may make sense to create t-shirt size widths (base, large, small) that can be applied to particular input archetypes (telephone, zip, names, address, age, etc.).